gfp12.go

```diff
@@ -174,6 +213,80 @@ func (e *gfP12) Square(a *gfP12) *gfP12 {
 	return e
 }
 
+// Special squaring loop for use on elements in T_6(gfP2) (after the
+// easy part of the final exponentiation. Used in the hard part
+// of the final exponentiation. Function uses formulas in
+// Granger/Scott (PKC2010).
+func (e *gfP12) SpecialSquare(a *gfP12) *gfP12 {
```

Comment:
it can be renamed to `SquareCyclotomic` or something more shorter. Similarly for the exponentiation by `u`

Reply:
renamed to SquareCyclo6